backend/kernelCI_app/views/hardwareDetailsView.py

```diff
@@ -175,7 +183,7 @@ def pass_in_filters(self, data, filters):
 
     def sanitize_records(self, records, selected_trees, filters):
         processed_builds = set()
-        builds = {"items": [], "issues": {}}
+        builds = {"items": [], "issues": {}, "failedWithUnkownIssues": 0}
```

Comment:
`failedWithUnkownIssues` is related to `tests` and `boots` not builds

Reply:
Done